frontend/src/scenes/events/createActionFromEvent.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { router } from 'kea-router'
+import api from 'lib/api'
+import { toast } from 'react-toastify'
+import { eventToName } from 'lib/utils'
+
+export function recurseSelector(elements, parts, index) {
```

Comment:
I'm not completely sure if it works and maybe you've considered this already but would it make sense/be possible to latch onto classnames rather than just id/tag_name? For example, the cypress inspector always uses classes. This might be not much better in some cases where pages heavily reused classes without id/names but it might be a bit more specific
<img width="554" alt="Screen Shot 2020-06-08 at 12 35 02 PM" src="https://user-images.githubusercontent.com/13127476/84056782-b541ff80-a984-11ea-8a19-c2406e275044.png">


In cases like this you might end up just matching a ton more than intended and some of these elements might have classes somewhere in the tree of elements
<img width="1392" alt="Screen Shot 2020-06-08 at 12 27 14 PM" src="https://user-images.githubusercontent.com/13127476/84056513-51b7d200-a984-11ea-81be-f490fe370ec3.png">

Reply:
Hm yeah good point. The annoying thing is classes can be auto generated or can indicate state (`.active` for example) that you don't want to include in a selector.

An alternative is adding the nth-child to all of it, as we do have that information.

Also, the selector only gets used if there's no href or text so it doesn't get used for everything.

What do you think?

Reply:
Ah fair point with the classes. The nth-child should work and yeah this would be a last possible case

Reply:
Added a ticket to leave it up to you as to whether you'll include it in this ticket or followup so this feature can get pushed. #963